ddtrace/internal/peer_service/__init__.py

```diff
@@ -0,0 +1,15 @@
+import functools
```

Comment:
I created a new module under internal with the assumption that we're trying to move away from a global Config and to per-product/per-feature modules.  Let me know if this should reside somewhere else

Reply:
This directory feels a bit off to me. "Peer service" is not really a product. It's more a tracing configuration. I think this configuration should be defined with the the rest of the tracing configurations (in `ddtrace.setting.config.py`). We can refactor this module to split up products in the future 🤞

Reply:
Can we put this feature in the `schema` directory?